spec/features/staff/users_can_create_a_planned_disbursement_spec.rb

```diff
@@ -21,17 +21,35 @@
       expect(page).to have_content t("action.planned_disbursement.create.success")
     end
 
-    scenario "the current financial quarter and year are pre selected" do
-      project = create(:project_activity, :with_report, organisation: user.organisation)
-      first_quarter_2019_2020 = "2019-04-01".to_date
+    context "when we are in the first quarter" do
+      scenario "the current financial quarter and year are pre selected" do
+        first_quarter_2019_2020 = "2019-04-01".to_date
+
+        travel_to first_quarter_2019_2020 do
```

Comment:
FYI I recently added the `travel_to_quarter(quarter, year)` spec helper, but I think your descriptive variable name works equally well here :)